### PR TITLE
Inverting FTDI for R9 serial access

### PR DESCRIPTION
**Pull request by tmcadam**
Added some notes on inverting FTDI for R9 serial access

### CODE REVIEW

**Comment by olliw42** [comment]
not a comment on this PR, but a general one. I think the R9 docu is starting to get useless ... way too many options and possibilities for working around the issues and flashing. IMHO we enter the territory where all open source projects start to fall appart and become largely useless, where only few guys understand all options and any new user is totally overwhelmed, and will shy away. I think we are in the course of crossing that clipping point and starting to achieve the opposite. We couldn't even get an experienced ArduPilot user getting along easily with all the setup ... 
I guess we are close to a point where we need either to separate off a "stable" project and docu from a "grow wild", or refrain from doing and supporting and documenting anything which is possible. 
When I started more than two years with this, reaching this kind of situation was what I didn't wanted to happen, as I stated it in Jan 2022: "_in my observation, projects which want to be everything tend to fail or get so complicated that they are rather failures_"  ... it appears I didn't do very well :(

**Comment by tmcadam** [comment]
I'm happy

File: docs/FRSKY_R9.md [hunk 1]
```diff
@@ -58,6 +60,14 @@ There are several DIY approaches for building an inverter dongle. A common appro
 
 <img src="images/frsky-max3232-inverter-scheme.jpg" width="360px">
 
+### Invert RX/TX on FTDI Serial Adapter ###
+
+This may not work on all versions of this adapter, many have non-genuine chips. Using the FT_prog tool it is simple to reconfigure these common serial adapters so that the RX and TX use inverted logic. THe tool can be downloaded [here](https://ftdichip.com/utilities/#ft_prog). Instructions on using the tool can be [found on GitHub](https://github.com/kaack/elrs-joystick-control?tab=readme-ov-file#configuring-the-ftdi-adapter). 
+
+#### DO NOT CONNECT THE 5V PIN! Only connect Gnd, RX, TX. ####
```
Comment (olliw42) [inline docs/FRSKY_R9.md:67]: that's for chapters not to emphasize. :)

File: docs/FRSKY_R9.md [hunk 2]
```diff
@@ -58,6 +60,14 @@ There are several DIY approaches for building an inverter dongle. A common appro
 
 <img src="images/frsky-max3232-inverter-scheme.jpg" width="360px">
 
+### Invert RX/TX on FTDI Serial Adapter ###
+
+This may not work on all versions of this adapter, many have non-genuine chips. Using the FT_prog tool it is simple to reconfigure these common serial adapters so that the RX and TX use inverted logic. THe tool can be downloaded [here](https://ftdichip.com/utilities/#ft_prog). Instructions on using the tool can be [found on GitHub](https://github.com/kaack/elrs-joystick-control?tab=readme-ov-file#configuring-the-ftdi-adapter).
```
Comment (olliw42) [inline docs/FRSKY_R9.md:65]: spelling, THe

Comment (jlpoltrack) [reply docs/FRSKY_R9.md:65]: Fixed

**Pull request merged**